test/test_bounds.cpp

```diff
@@ -41,7 +41,7 @@ constexpr Real T_split = 1e4;
 constexpr int N_per_decade_fine = 200;
 constexpr Real N_factor = 5;
 
-constexpr Real REAL_TOL = std::numeric_limits<Real>::epsilon() * 1e3;
+constexpr Real REAL_TOL = std::numeric_limits<Real>::epsilon() * 1e4;
```

Comment:
Why the change here? If it was for NQT_O2, I thought that would have promoted a stricter tolerance?

Reply:
The sqrt in fast pow appears to introduce error at the level of roundoff, since sqrt is itself transcendental. Previously the fastpow was all algebraic primitives.